x/outline-connectivity-app/.gitignore

```
@@ -0,0 +1,7 @@
+output
+node_modules
+.yarn/cache
+.yarn/sdks
+.yarn/unplugged
+.yarn/install-state.gz
```

This seems to be [Yarn's recommendation](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored):

```suggestion
.pnp.*
.yarn/*
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```